package.json

```diff
@@ -19,9 +19,11 @@
     "build": "webpack"
   },
   "dependencies": {
+    "@types/uuid": "^3.4.2",
```

Comment:
I believe @types are only required for development, so this can be a dev-dependency.